src/net_processing.cpp

```diff
@@ -2279,6 +2289,29 @@ void PeerManagerImpl::RelayInv(CInv &inv, const int minProtoVersion)
     });
 }
 
+void PeerManagerImpl::RelayDSQ(const CCoinJoinQueue& queue)
+{
+    CInv inv{MSG_DSQ, queue.GetHash()};
+    std::vector<NodeId> nodes_send_all;
+    {
+        LOCK(m_peer_mutex);
+        for (const auto& [nodeid, peer] : m_peer_map) {
+            if (peer->m_wants_dsq == Peer::WantsDSQ::INV) {
+                PushInv(*peer, inv);
+            } else if (peer->m_wants_dsq == Peer::WantsDSQ::ALL) {
+                nodes_send_all.push_back(nodeid);
+            }
```

Comment:
let's use switch/case to make sure we process all possible `Peer::WantsDSQ` values
```suggestion
            switch (peer->m_wants_dsq) {
            case Peer::WantsDSQ::NONE:
                break;
            case Peer::WantsDSQ::INV:
                PushInv(*peer, inv);
                break;
            case Peer::WantsDSQ::ALL:
                nodes_send_all.push_back(nodeid);
                break;
            }
```

Reply:
apples in latest force-push